deepsource/issues/queries/list_file_issues.go

```diff
@@ -102,9 +102,8 @@ func (f FileIssuesListRequest) Do(ctx context.Context, client IGQLClient) ([]iss
 	}
 
 	// Formatting the query response w.r.t the output format of the SDK as specified in `issues_list.go`
-	var issuesData []issues.Issue
+	issuesData := make([]issues.Issue, len(respData.Repository.File.Issues.Edges))
```

Comment:
Used `make` to initialize `issueData`, as currently it panics due to it being `nil`.

Reply:
Understood. Thanks!